### ISSUE TITLE
`Climate` data column names are incorrect.

### ISSUE THREAD

**Issue by SimonGoring** [gavinsimpson/analogue#17]
Using `analogue` version 0.17-0:

```r
> data(Climate)
> Climate[1,]
```
Returns:
```r
  Sitename   lon   lat  tapr tmay tjun tjul taug tsep toct tnov  tdec pjan pfeb pmar papr
1    -23.1 -23.6 -20.5 -13.1   -4  1.5  5.8  5.6  2.2 -3.1 -8.7 -17.9   21   19   19   22
  pmay pjun pjul paug psep poct pnov pdec      tave tmax  tmin  gdd0 gdd5  mtco mtwa annp
1   25   43   63   59   43   40   33   23 -8.241667 25.8 -45.4 427.2 25.6 -23.6  5.8  410
```

It looks like the tjan, tfeb, tmar are replaced by Sitename, lon and lat.

**Comment by gavinsimpson** [comment]
Well that's just silly of me isn't it!

Thanks @SimonGoring I'll grab the latest version of the data and prepare a new data object.

**Comment by gavinsimpson** [comment]
Thanks again @SimonGoring for  spotting this: it is now fixed and updated to the 1-7.3 version of the NAMPD from Konrad's website.

Some inconsistencies are creeping in with the multiple locations for the data (Jack's site only has version 1-7.2) and some of the metadata is getting a little out of sync with the data files. I'll follow up with Jack and Konrad about this.
